specs/waku/v2/waku-filter.md

```diff
@@ -1,28 +1,34 @@
 ---
 title: Waku
-version: 2.0.0-alpha5
-status: Raw
-authors: Oskar Thorén <oskar@status.im>
+version: 2.0.0-beta1
+status: Draft
+authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
 ---
 
 # Table of Contents
 
-TODO
+- [Abstract](#abstract)
+- [Content filtering](#content-filtering)
+  * [Rationale](#rationale)
+  * [Protobuf](#protobuf)
+- [Changelog](#changelog)
+- [Copyright](#copyright)
+- [References](#references)
 
 # Abstract
 
-Filter spec.
+`WakuFilter` is a protocol that enables subscribing to messages that a peer receives. This is a more lightweight version of `WakuRelay` specifically designed for bandwidth restricted devices.
```

Comment:
Receives where? subscribing to what messages? "that match a filter on the content"

Reply:
Why is it more lightweight? What makes it designed for BW restricted devices?

What I wrote were just notes for inspiration of what to write, not actually what should be written verbatim